### ISSUE TITLE
Redesign Call to Action

### ISSUE THREAD

**Issue by NickEngmann** [opensandiego/p2p-lending#26]
Current Call to Action looks kind of weird, open to any ideas for a redesign

How it currently looks:
<img width="592" alt="Screen Shot 2019-05-23 at 11 21 46 PM" src="https://user-images.githubusercontent.com/1075072/58306809-a1dfe880-7db1-11e9-8902-9b6f67bcc73a.png">

**Comment by Mic-J-Lee** [comment]
Does it look like this now, or is that just me 😆
![image](https://user-images.githubusercontent.com/21283857/58338303-628ab980-7dfc-11e9-9bf0-80e074479c6f.png)

**Comment by Mic-J-Lee** [comment]
Does it look like this now, or is it just me 😆
![image](https://user-images.githubusercontent.com/21283857/58338363-8e0da400-7dfc-11e9-93a4-737362005428.png)

**Comment by NickEngmann** [comment]
If you pull the most recent changes off the master branch and run the web app locally (using `npm run start` in the /p2plending/frontend folder) you should be seeing the first example.

**Comment by Mic-J-Lee** [comment]
Oh gotcha. I'd pulled already, My mistake was going to localhost:8000. I didn't expect there to be a front end view from the python backend, how interesting!

**Comment by GreatestDane** [comment]
Are you still looking for a redesign?

**Comment by NickEngmann** [comment]
Yes, still looking for a button redesign and probably an entire site redesign. If you have the time you should definitely post some designs 
![bitmoji](https://render.bitstrips.com/v2/cpanel/37d6cfa2-2482-4546-a129-80257de14ebd-f35a46c4-b1dd-4da0-a9de-3b0fa0f211ba-v1.png?transparent=1&palette=1&width=246)

**Comment by jamesravenscroft** [comment]
Hi, I could help with this.
